packages/core/src/utils/filter.ts

```diff
@@ -18,6 +18,29 @@
 
 import {Color} from '@axelor/aos-mobile-ui';
 
+interface Anomaly {
+  version: number;
+  message: string;
+  checkType: string;
+  modelName: string;
+}
+
+export function filterAnomaly(anomalyList: Anomaly[]) {
+  const BEFORE = -1;
+  const EQUAL = 0;
+  const AFTER = 1;
+
+  return anomalyList.sort((a: Anomaly, b: Anomaly) => {
+    if (a.checkType === b.checkType) {
+      return EQUAL;
+    } else if (a.checkType === 'error' && b.checkType === 'alert') {
```

Comment:
Those types 'error' & 'alert' are used more than one time, please create helper class in `types` folder to contain type values, color depending on type, sort function etc